### ISSUE TITLE
version mismatch in letsencrypt/__init__.py

### ISSUE THREAD

**Issue by floe** [letsencrypt/letsencrypt#2019]
I'm currently running the 0.1.1 tag, and it seems that in that branch, `__version__ = '0.2.0.dev0'`, leading to mismatches with acme-0.1.1.

Following trivial patch fixes things for me:

```diff
diff --git a/letsencrypt/__init__.py b/letsencrypt/__init__.py
index 1c7815f..e011c3f 100644
--- a/letsencrypt/__init__.py
+++ b/letsencrypt/__init__.py
@@ -1,4 +1,4 @@
 """Let's Encrypt client."""
 
 # version number like 1.2.3a0, must have at least 2 parts, like 1.2
-__version__ = '0.2.0.dev0'
+__version__ = '0.1.1'
```

**Comment by willysr** [comment]
Duplicate of my issue: https://github.com/letsencrypt/letsencrypt/issues/1966

**Comment by kuba** [comment]
While it is duplicate, previous issue was close without fixing: there is still no tag for the code base used for 0.1.1 release...

**Comment by willysr** [comment]
I thought the fix was in the release scripts: https://github.com/letsencrypt/letsencrypt/commit/42333536517329dcb6584bf9da9c52389ff1be27

it will be solved by itself when developers released 0.1.2 or 0.2.0

**Comment by kuba** [comment]
No, releasing completely new version doesn't magically create a proper version tag for previous release.

**Comment by pde** [comment]
OK, I've uploaded the [v0.1.1-corrected](https://github.com/letsencrypt/letsencrypt/releases/tag/v0.1.1-corrected) tag, which should match the actual release uploaded to PyPI.

**Comment by kuba** [comment]
https://github.com/letsencrypt/letsencrypt/releases/tag/v0.1.1-corrected points at edd20a8 (were some version strings are still broken) rather than 9d50c3e.... I think you should now create `v0.1.1-corrected-corrected`...
